README.md

```diff
@@ -10,15 +10,17 @@ implement user experiences consistent with
 <a href="https://lerna.js.org">
   <img src="https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg" alt="Maintained with Lerna" />
 </a>
-[![Release](https://github.com/Workday/canvas-kit/actions/workflows/release.yml/badge.svg)](https://github.com/Workday/canvas-kit/actions/workflows/release.yml)
+<a href="https://github.com/Workday/canvas-kit/actions/workflows/release.yml">
+  <img alt="Release" src="https://github.com/Workday/canvas-kit/actions/workflows/release.yml/badge.svg">
+</a>
```

Comment:
I noticed this badge was rendering fine in Storybook but not in our Readme. So I updated it to use HTML instead of MD.